src/pages/Products/Components/Card.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './Card.scss';
+import { Link } from 'react-router-dom';
+
+class Card extends Component {
+  render() {
+    const { list } = this.props;
+    const tagArr = list.tag.map(item => item.tag);
+    const showTagList = tagArr.map(tag => {
+      return <span className={tag}>{tag}</span>;
+    });
+    const showSoldOut = <span className="soldOut">Sold Out</span>;
+
+    return (
+      <Link to={`/product/${list.id}`}>
+        <li className="card" data-num={list.id}>
+          <img
+            src={list.imgUrl}
+            alt={list.imgAlt}
+            className={!list.stock && 'opacity'}
```

Comment:
```suggestion
            className={list.stock ? '' : 'opacity'}
```
list.stock이 true인 경우 img 태그의 클래스명이 `null`이라는 문자열로 출력될 것 같습니다.

Reply:
아 그렇네요! 수정반영되었습니다.